emissions.md

```diff
@@ -121,11 +121,30 @@ where:
 
 <br/>
 
-Non-CO<sub>2</sub> GHG emissions are proportional to the activity except for any reductions in emission intensity due to the MAC curve. As noted above, the MAC curves are assigned to a wide variety of technologies, mapped directly from [EPA 2019](#epa2019). Under a carbon policy, emissions are reduced by an amount determined by the MAC curve.
+
+`MAC`-related parameters:
+
+XML Tag | Description |
+| :------------- |:-------------|
+| `tech-change` | annual improvement of reduction potential defined in MAC curve|
+| `mac-phase-in-time` | MAC phase-in periods (see description below)|
+
+<br/>
+
+Non-CO<sub>2</sub> GHG emissions are proportional to the activity except for any reductions in emission intensity due to the MAC curve. As noted above, the MAC curves are assigned to a wide variety of technologies, mapped directly from [EPA 2019](#epa2019) ([Ou et al. 2021](#ou2021a)). Under a carbon policy, emissions are reduced by an amount determined by the MAC curve.
 
 The default set-up is that MAC curves use the scenario's carbon price (if any). The non-CO<sub>2</sub> GHG MACs are an exogenous input, and are read in as the percent of emissions abated as a function of the emissions prices. Note that they are read in with explicit cost points (i.e., piece-wise linear form), with no underlying equation describing the percentage of abatement as a function of the carbon price.
 
-Below-zero (i.e. “no cost”) MAC mitigation (e.g. MAC reduction percentage is > 0 at zero carbon price) are applied in the reference case and phased in over several decades. (This can be turned off by setting the no-zero-cost-reductions option to 1 within a MAC curve.)
+The parameter technological change (`tech-change`) represents annual improvement of reduction potential at all prices (essentially shifting the entire MAC curve to the right). For non-CO2 GHGs, MAC is typically defined in the initial available year (typically 2025), while tech.change starts from the next modeling period (i.e. 2030) to adjust the base-year MAC. `tech-change` from 2030 to 2050 is backward calculated from the maximum mitigation potential in [EPA 2019](#epa2019), and tech.change after 2050 is assumed to be the average of pre-2050 tech.change (since current EPA mitigation report only contains mitigation potential till 2050) ([Ou et al. 2021](#ou2021a)). 
+
+$$
+R_{t_2, p}=R_{t_1, p}*(1+TC_{t_2})^{t_2-t_1}
+$$
+where $R_{t_1, p}$ and $R_{t_2, p}$ represent MAC reduction in year $t_1$ and $t_2$ at emission price $p$, respectively. $TC_{t_2}$ is the tech.change in $t_2$.  
```

Comment:
The equations in the where need to be surrounded by double `$$` instead of just one `$` to render